src/substrate-bridge.md

```diff
@@ -0,0 +1,187 @@
+# Substrate bridge
+
+## Main components
+
+SORA 2 interaction with Polkadot/Kusama and other relay chains is visualised in the following diagram:
+
+![](./assets/substrate-bridge-components-svg.svg)
+
+<details>
+  <summary>UML diagram code</summary>
+
+```
+
+@startuml
+
+skinparam component {
+  BorderColor white
+  BackgroundColor red
+}
+
+skinparam interface {
+  BorderColor black
+  BackgroundColor red
+}
+
+Component [SORA mainnet]
+interface "Federated\nbridge" as FB
+
+cloud {
+  [SORA parachain] as SP
+  [Relay chain\n(Polkadot/Kusama)] as RC
+  interface "XCM" as XCM1
+  interface "XCM" as XCM2
+  [Other parachains] as OP
+}
+
+[SORA mainnet] - FB
+FB - SP
+SP - XCM1
+XCM1 - RC
+RC - XCM2
+XCM2 - OP
+@enduml
+
+```
+
+</details>
+
+Let's go through all the components mentioned in the figure:
+
+### SORA mainnet
+
+A standalone network developed on top of the Substrate blockchain is the main object of this documentation. It has all the incoming and outgoing messages to other systems crossing the **Federated Bridge**, which is a central gateway to an external ecosystem.
+
+### Federated Bridge
+
+You can read about the Federated Bridge in [Interoparibility](./interoperability).
+
+### SORA parachain
+
+The SORA Parachain serves as a blockchain gateway, allowing seamless integration between SORA and any relay chain ecosystem. It facilitates automatic token transfers, eliminating the need for users to send tokens via extrinsics on the SORA Parachain. By design, the SORA Parachain eliminates the requirement for additional transaction signing. The Parachain maintains a comprehensive record of tokens that can be transferred, with registration information stored in the XCMApp pallet. This information includes the mapping between SORA Mainnet AssetId and XCM Multilocation for each token.
```

Comment:
```suggestion
The SORA parachain serves as a blockchain gateway, allowing seamless integration between SORA and any relay chain ecosystem. It facilitates automatic token transfers, eliminating the need for users to send tokens via extrinsics on the SORA parachain. 
By design, the SORA parachain eliminates the requirement for additional transaction signing. The parachain maintains a comprehensive record of tokens that can be transferred, with registration information stored in the XCMApp pallet. This information includes the mapping between the SORA mainnet AssetId and XCM Multilocation for each token.
```

Reply:
Just like network is not part of the name in SORA network (lowcase), we should write SORA parachain